Example/Sources/Base.lproj/LaunchScreen.storyboard

```diff
@@ -22,7 +22,7 @@
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <imageView userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="mklogo" translatesAutoresizingMaskIntoConstraints="NO" id="NHX-yN-7Jk">
-                                <rect key="frame" x="76" y="99" width="222" height="127"/>
+                                <rect key="frame" x="76.5" y="99" width="222" height="127.5"/>
```

Comment:
Maybe a nitpick and not sure but I guess this indicates the rect is misaligned.
Open a simulator and tap `Debug` → `Color Misaligned Images` on the menu bar would tell you if it's misaligned.
Misaligned image itself is not a problem but causes performance losses on particular devices.

Reply:
Thank you for sharing! I will check 👍